modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -669,16 +669,18 @@ private FileEntry _getFileEntry(
 				return scope;
 			});
 		fileEntry.setThumbnailURL(
-			() -> {
-				String thumbnailURL = _dlURLHelper.getThumbnailSrc(
-					new LiferayFileEntry(dlFileEntry), null);
+			() -> (String)NestedFieldsSupplier.supply(
```

Comment:
Why this cast?

Reply:
There is no reason, I 've just delete it.

thanks @adolfopa